allennlp/data/data_loaders/multitask_data_loader.py

```diff
@@ -273,6 +277,13 @@ def __init__(self, inner: DatasetReader, head: str, **kwargs):
         self.inner = inner
         self.head = head
 
+    def _set_worker_info(self, info: Optional[WorkerInfo]) -> None:
+        """
+        Should only be used internally.
+        """
+        self._worker_info = info
+        self.inner._worker_info = info
```

Comment:
Just in case `DatasetReader._set_worker_info()` is modified in the future, let's do this:

```suggestion
        super()._set_worker_info(info)
        self.inner._set_worker_info(info)
```